### PR TITLE
Upgrade jackson and vertx to upgrade snakeyaml to 2.0

### PR DESCRIPTION
**Pull request by siladu**

### CODE REVIEW

**Review by usmansaleem — APPROVED** [review]
LGTM

File: gradle/versions.gradle [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 dependencyManagement {
   dependencies {
-    dependency 'com.fasterxml.jackson.core:jackson-databind:2.14.2'
-    dependency 'com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.14.2'
+    dependency 'com.fasterxml.jackson.core:jackson-databind:2.15.0-rc3'
```
Comment (jframe) [inline gradle/versions.gradle:16]: Normally rather not use an RC but in this case fixing a CVE think it's worth it

**Review by jframe — APPROVED** [review]
LGTM

File: gradle/versions.gradle (hunk 1 above)
Comment (siladu) [reply gradle/versions.gradle:16]: Created an issue as a reminder: https://github.com/ConsenSys/web3signer/issues/757

**Pull request merged**